tests/all-package-defaults.nix

```diff
@@ -56,6 +63,14 @@ let
   ++ lib.optionals (hostPlatform.isDarwin && hostPlatform.isAarch64) [
     # As of 2025-07-25, zig-zlint is failing on aarch64-darwin
     "zig-zlint"
+
+    # 2025-09-08, build failure
+    # https://github.com/NixOS/nixpkgs/pull/441058
+    "verible"
+
+    # 2025-09-08 ttfautohint hangs forever
+    "texlive-combined-medium"
+    "texlive"
```

Comment:
Should we just move the other texlive entries to the `isDarwin` group, instead of listing them twice?

Reply:
I guess disabling it separately means we can more easily re-enable it separately, if one platform starts building again.